prow/tide/tide.go

```diff
@@ -1684,10 +1684,11 @@ func sortPools(pools []Pool) {
 }
 
 type subpool struct {
-	log    *logrus.Entry
-	org    string
-	repo   string
-	branch string
+	log      *logrus.Entry
+	org      string
+	repo     string
+	cloneURI string
```

Comment:
It doesn't look like this field is ever populated. Please fix and add a unit test.

Reply:
it's unused for GitHub, meant to add it in the next PR